src/styles/overrides/_se-form.scss

```diff
@@ -0,0 +1,92 @@
+.seform {
+  form {
+    .seText label {
+      @extend .dg_label;
+    }
+
+    fieldset {
+      @extend .dg_fieldset;
+
+      legend {
+        @extend .dg_legend;
+      }
+
+      .seRadioLabel {
+        @extend .dg_label;
+        @extend .dg_radio-label;
+      }
+
+      .seCheckboxLabel {
+        @extend .dg_label;
+        @extend .dg_checkbox-label;
+      }
+
+      .seRadio {
+        @extend .dg_radio-input;
+      }
+
+      .seCheckbox {
+        @extend .dg_checkbox-input;
+      }
+
+      .seLabelCell {
+        @extend .dg_label;
+        margin-top: 0;
+        margin-bottom: 5px;
+
+        .seRequiredMarker {
+          margin-left: 5px;
+        }
+      }
+
+      .SEAFGroupHorizontal,
+      .SEAFLabelHorizontal,
+      .SEAFGroupVertical,
+      .SEAFLabelVertical {
+        margin-bottom: $default-margin;
+      }
+    }
+
+    .seRequiredMarker {
+      color: $red;
+      font-size: 0;
+      font-weight: 600;
```

Comment:
I would use the variable` $font-medium`.

Reply:
this has been addressed.